src/compare.py

```diff
@@ -144,6 +144,17 @@ def stop(status):
 # This will let the camera adjust its light levels while we're importing for faster scanning
 video_capture.grab()
 
+# Disable autoexposure
+exposure = config.getint("video", "exposure", fallback=-1)
+if exposure != -1:
+	# For a strange reason on some cameras (e.g. Lenoxo X1E)
+	# setting manual exposure works only after a couple frames
+	# are captured.
+	for i in range(5):
+		video_capture.grab()
```

Comment:
I don't really like this delay. Could you possibly use the main loop, which also counts frames, to check and run this?